jax/_src/xla_bridge.py

```diff
@@ -247,6 +248,29 @@ def make_cpu_client() -> xla_client.Client:
 )
 
 
+def _check_cuda_compute_capability(devices_to_check):
+    try:
+      from jaxlib.gpu_triton import get_compute_capability
+    except ImportError as e:
+      logger.warning(
```

Comment:
I don't think we should emit a warning. 

`get_compute_capability` can fail to import either because CPU jaxlib is installed, and thus this check is irrelevant, or if the jaxlib version is too old. The oldest support jaxlib (0.4.18) does have `get_compute_capability` so we are safe to assume it always exists in a GPU jaxlib.